cmd/kubeadm/app/constants/constants.go

```diff
@@ -28,4 +28,9 @@ const (
 	APIServerKubeletClientCertAndKeyBaseName = "apiserver-kubelet-client"
 	APIServerKubeletClientCertName           = "apiserver-kubelet-client.crt"
 	APIServerKubeletClientKeyName            = "apiserver-kubelet-client.key"
+
+	AuthzModeAlwaysAllow = "AlwaysAllow"
```

Comment:
You're right; it's not used anywhere, but I guess it might be there for completeness, since we were told to copy these constants from another package (which also have these constants). They were copied over here to kubeadm until a subpackage is made for the authorizer that includes only the constants.